### PR TITLE
Added Date/Time module and pipeline

### PR DESCRIPTION
**Pull request by nitinvwaran**
Created and configured date/time module in the pipeline. Tested on nearly the full amalgum corpus, with some files skipped (~10 files) due to previous errors in the pipeline (stanfordnlp error for reddit_511 and reddit_799 during POS Tagging, and memory issues with 8-9 files , most of them large Reddit files, during dependency parse step)

Date/Time module is a pipeline consisting of HeidelTime, a Random Forest Model to eliminate FPs from HeidelTime, and a custom normalization function to convert the TIMEX3 attributes to TEI-style date/time attributes. The precision/recall/F1 on the GUM corpus v6 with this pipeline is as below, for expression detection only:

Hypothesis | True Positives | False Positives | False Negatives | Precision | Recall | F1Score
fuzzy | 886 | 17 | 182 | 0.981173864894795 | 0.829588014981273 | 0.899036022323693
exact | 810 | 126 | 237 | 0.865384615384615 | 0.773638968481375 | 0.816944024205749

### CODE REVIEW

File: nlp_controller.py [hunk 1]
```diff
@@ -34,17 +35,18 @@
...
+    "rst_parser": RSTParser
 }
 
 
 class NLPController:
     def __init__(self, opts):
         logging.info("Initializing NLP Controller...")
         opts.update(
-            {"SCRIPT_DIR": SCRIPT_DIR, "LIB_DIR": LIB_DIR, "BIN_DIR": BIN_DIR, "TT_PATH": TT_PATH,}
+            {"SCRIPT_DIR": SCRIPT_DIR, "LIB_DIR": LIB_DIR, "BIN_DIR": BIN_DIR, "TTG_PATH": TTG_PATH}
```
Comment (lgessler) [inline nlp_controller.py:49]: could you update our two existing TT modules to handle this name change? There's a [tokenizer](https://github.com/gucorpling/amalgum/blob/master/nlp_modules/tt_tokenizer.py) and a [tagger](https://github.com/gucorpling/amalgum/blob/master/nlp_modules/tt_tagger.py)

Comment (lgessler) [reply nlp_controller.py:49]: actually, looks like only the tagger needs to know where the binary is

**Review by lgessler — CHANGES_REQUESTED** [review]

File: nlp_controller.py (hunk 1 above)
Comment (nitinvwaran) [reply nlp_controller.py:49]: changed the paths in the tree tagger module so new binary is discoverable. Note that the binary is downloaded only during date/time processing, which is a later step in the pipeline.

**Review by lgessler — APPROVED** [review]

**Comment by lgessler** [comment]
Awesome! We should do another release of the corpus now that this is in, but for now we're blocked on #9

**Pull request merged**